proto/cosmos/group/v1/tx.proto

```diff
@@ -306,11 +302,16 @@ message MsgSubmitProposal {
   // whether it should be executed immediately on creation or not.
   // If so, proposers signatures are considered as Yes votes.
   Exec exec = 5;
+
+  // title is the title of the proposal.
+  string title = 6;
+
+  // summary is the summary of the proposal.
+  string summary = 7;
```

Comment:
I was thinking of reusing the same max length as metadata. I did the check on summary but can add to title. Went with metadata length because it shouldn't be long

Reply:
> Went with metadata length because it shouldn't be long

I think I didn't follow the whole conversation for these changes, so if there was already consensus on this, then I can approve.

But my concern is: `summary` is not the long description, right? Where would summary be shown/used btw? Also, iiuc the initial problem was that without these 2 news fields, it breaks explorers (for gov). But if we introduce a `summary` with a 255-char lenghth, it still break explorers, right? e.g. https://www.mintscan.io/cosmos/proposals/93, the "Details" field would still need to be resolved with IPFS.